include/bitcoin/bitcoin/utility/png.hpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2011-2015 libbitcoin developers (see AUTHORS)
+ *
+ * This file is part of libbitcoin.
+ *
+ * libbitcoin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License with
+ * additional permissions to the one published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version. For more information see LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef LIBBITCOIN_PNG_HPP
+#define LIBBITCOIN_PNG_HPP
+
+#include <cstdint>
+#include <istream>
+#include <bitcoin/bitcoin/compat.hpp>
+#include <bitcoin/bitcoin/define.hpp>
+#include <bitcoin/bitcoin/utility/data.hpp>
+
+#ifdef WITH_PNG
+#include "png.h"
+#endif
+
+namespace libbitcoin {
+
+#ifdef WITH_PNG
+
+namespace png {
```

Comment:
For the sake of more consistent organization we have moved to using class statics as opposed to global functions. I recommend converting the `bc::png` namespace to a class, exposing these constants as static constant members, and the below functions as static methods.